examples/question-answering/run_squad.py

```diff
@@ -319,7 +319,7 @@ def evaluate(args, model, tokenizer, prefix=""):
             eval_feature = features[feature_index.item()]
             unique_id = int(eval_feature.unique_id)
 
-            output = [to_list(output[i]) for output in outputs]
+            output = [to_list(output[i]) for output in outputs.to_tuple()]
```

Comment:
Indexing with an integer is normally possible with the `ModelOutput`. Why is this change necessary?

Reply:
The example was raising an indexing error here once I updated `self.return_dict` to `True` in the `PretrainedConfig` class.